### PR TITLE
docs: Add doorbell example

### PR DESCRIPTION
**Pull request by felipecrs**

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -1215,3 +1215,9 @@ automations:
 ```
 
 ![Zoom automation example](images/zoom-automation.gif 'Zoom automation example :size=400')
+
+## Doorbell
```
Comment (dermotduffy) [inline docs/examples.md:1219]: The examples are intended to be sorted alphabetically. Mind moving it up? Thank you!

Comment (felipecrs) [reply docs/examples.md:1219]: Ops, got it. Will move.

File: docs/examples.md [hunk 2]
```diff
@@ -1183,6 +1183,12 @@ overrides:
           y: 100
 ```
 
+## Doorbell
```
Comment (dermotduffy) [inline docs/examples.md]:
```suggestion
### Doorbell
```

Comment (felipecrs) [reply docs/examples.md]: It was actually a mistake, please check again.

File: docs/examples.md [hunk 3]
```diff
@@ -291,6 +291,12 @@ elements:
           - camera.office
 ```
 
+## Doorbell
+
+[felipecrs/dahua-vto-on-home-assistant](https://github.com/felipecrs/dahua-vto-on-home-assistant#readme) provides an example on how this card can be used to answer a doorbell in Home Assistant.
+
+![Doorbell example](https://github.com/user-attachments/assets/39fa9213-ce32-45f4-a236-b6a99797a217 'Doorbell example :size=400')
```
Comment (dermotduffy) [inline docs/examples.md:298]: Maybe put this in the `docs/images` directory directly so the docs are hermetic?

**Comment by dermotduffy** [comment]
Thanks @felipecrs !

**Pull request merged**